lib/src/integrationtest.rs

```diff
@@ -0,0 +1,26 @@
+//! Module used for integration tests; should not be public.
+
+use std::path::Path;
+
+fn is_container() -> bool {
+    // https://stackoverflow.com/questions/20010199/how-to-determine-if-a-process-runs-inside-lxc-docker
+    for p in ["/run/.containerenv", "/.dockerenv"] {
```

Comment:
OK this was fun.  I wish I could add a comment on https://stackoverflow.com/questions/20010199/how-to-determine-if-a-process-runs-inside-lxc-docker but apparently I don't have enough reputation.